modules/react/select/stories/Select.stories.mdx

```diff
@@ -34,79 +34,145 @@ yarn add @workday/canvas-kit-react
 
 ### Basic Example
 
-Select should be used in tandem with [Form Field](/components/inputs/form-field/) to meet
-accessibility standards.
+`Select` supports a
+[dynamic API](/getting-started/for-developers/resources/collection-api/#dynamic-items) where you
+pass an array of items via the `items` prop and provide a render function to display the items. The
+items may be provided as an
+[array of strings](/getting-started/for-developers/resources/collection-api/#array-of-strings) or an
+[array of objects](/getting-started/for-developers/resources/collection-api/#array-of-objects).
```

Comment:
[Your original Select docs](https://5d854c26ba934e0020f5e98a-bdjnqbgofl.chromatic.com/?path=/docs/components-inputs-select--basic), for reference.

Originally, you had a separate `Registering Items` section in the Select MDX with examples. I decided to move that content into the `Basic Example` and link out to the `Array of Strings` and `Array of Objects` sections in the Collection MDX. It seemed like that would be sufficient instead of repeating that content here.